<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button_post">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Post Status!</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_main_twit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="list_follower1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>171</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="list_follower2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>251</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Main Twit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Follower Twit 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Follower Twit 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="status_enter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
